README.md

```diff
@@ -1,84 +1,66 @@
-At time of writing [Travis-CI](https://travis-ci.org/) has [support for building Haskell packages](http://about.travis-ci.org/docs/user/languages/haskell/) but only for a single GHC configuration (i.e. *Haskell Platform 2012.2.0.0 with GHC 7.4.1*). By following this guide, you can set up [Travis-CI](https://travis-ci.org/) jobs which have access to the following GHC versions (all compiled for *Ubuntu Linux 12.04 LTS 64-bit*):
+This project provides a script generator for continuous-integration testing of Haskell packages. It currently provides substantive support for:
 
- - GHC 6.12.3,
- - GHC 7.0.1, GHC 7.0.2, GHC 7.0.3, GHC 7.0.4,
- - GHC 7.2.1, GHC 7.2.2,
- - GHC 7.4.1, GHC 7.4.2,
- - GHC 7.6.1, GHC 7.6.2, GHC 7.6.3,
- - GHC 7.8.1, GHC 7.8.2, GHC 7.8.3, GHC 7.8.4
- - GHC 7.10.1, GHC 7.10.2, GHC 7.10.3
- - GHC 8.0.1, GHC 8.0.2
- - GHC 8.2.1, GHC 8.2.2
- - GHC 8.4.1, GHC 8.4.2, GHC 8.4.3, GHC 8.4.4
- - GHC 8.6.1, GHC 8.6.2, GHC 8.6.3, GHC 8.6.4, GHC 8.6.5
- - GHC 8.8.1, GHC 8.8.2, GHC 8.8.3, GHC-8.8.4
- - GHC 8.10.1, GHC-8.10.2, GHC-8.10.3, GHC-8.10.4
- - GHC 9.0.1
- - GHC HEAD.
+- [Travis-CI](https://travis-ci.org/)
+- [GitHub Actions](https://docs.github.com/en/actions)
 
-Each GHC version is provided in a separate `ghc-<version>` `.deb` package installing into `/opt/ghc/<version>` (thus allowing to be installed at the same time if needed) published in a [PPA](https://launchpad.net/~hvr/+archive/ghc). The easiest way to "activate" a particular GHC version is to prepend its `bin`-folder to the `$PATH` environment variable (see example in next section).
+Features include:
 
-Note: For actually enabling continuous integration for a GitHub hosted project, see section [Getting Started](http://about.travis-ci.org/docs/user/getting-started/) in [Travis-CI](https://travis-ci.org/)'s online documentation.
+- Multiple [GHC](http://haskell.org/ghc) support
+- Building with specific constraints
+- Dependency caching
+- cabal and cabal.project support (see [Nix-style local builds documentation](https://cabal.readthedocs.io/en/latest/nix-local-build-overview.html))
+- Runs tests and builds benchmarks
+- Support for Haskell build tools, including [Haddock](https://hackage.haskell.org/package/haddock), [GHCJS](https://github.com/ghcjs/ghcjs) & [HLint](https://hackage.haskell.org/package/hlint)
+- doctest runner integration using either [doctest](https://hackage.haskell.org/package/doctest) or [cabal-docspec](https://github.com/phadej/cabal-extras/blob/master/cabal-docspec/MANUAL.md).
 
-### Quick-start instructions
+Usage
+---
 
-* Step 1: Clone and install this project in/from any directory
+See `haskell-ci --help` for details.
 
-    ```bash
-    $ git clone https://github.com/haskell-CI/haskell-ci.git
-    $ cd haskell-ci
-    $ cabal new-install haskell-ci:exe:haskell-ci
-    ```
+1. Install `haskell-ci`
 
-  or
+```bash
+cabal install haskell-ci
+```
```

Comment:
Maybe say which GHC versions are supported for building `haskell-ci` or recommend a specific version here. Compare https://github.com/haskell-CI/haskell-ci/issues/557.